topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,714 @@
+- What is Xarray?
+- How to manipulate Xarrays?
+- How to print metadata information?
+- How to make a selection?
+- How to visualize?
+- How to filter? 
+- How to make reduction operations (mean, max, min)?
+- How to resample my data?
+- Where to go next?
+objectives:
+- Learn to get metadata information using Xarray Galaxy Tools
+- Learn to select data
+- Learn to visualize data
+- Learn to filter, make reduction operations (mean, max, min)
+- Learn to resample my data
+- Learn to cite and contribute to Pangeo
+time_estimation: 1H
+key_points:
+- Pangeo ecosystem enables big data analysis in geosciences
+- Xarray is an important Python package for big data analysis in geosciences 
+- Xarray can be used to read, select, mask and plot netCDF data
+- Xarray can also be used to perform global operations such as mean, max, min or resampling data
+tags:
+  - pangeo
+  - interactive-tools
+  - jupyter-notebook
+contributors:
+- annefou
+notebook:
+  language: python
+  snippet: topics/climate/tutorials/pangeo-notebook/preamble.md
+---
+
+
+# Introduction
+{:.no_toc}
+
+In this tutorial, we will learn about [Xarray](https://xarray.pydata.org/), one of the most used Python library from the [Pangeo](https://pangeo.io/) ecosystem. 
+
+We will be using data from [Copernicus Atmosphere Monitoring Service](https://ads.atmosphere.copernicus.eu/)
+and more precisely PM2.5 ([Particle Matter < 2.5 μm](https://en.wikipedia.org/wiki/Particulates#Size,_shape_and_solubility_matter)) 4 days forecast from December, 22 2021. Parallel data analysis with Pangeo is not covered in this tutorial. 
+
+> ### {% icon comment %} Remark
+>
+> This tutorial uses data on a regular latitude-longitude grid. More complex and irregular grids are not discussed in this tutorial. In addition,
+> this tutorial is not meant to cover all the different possibilities offered by Xarrays but shows functionalities we find useful for day to day
+> analysis.
+>
+{: .comment}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Analysis
+
+## Import Python packages
+
+Some packages may need to be installed first. For example `cmcrameri` is missing, so we need to install it by entering the following command in a new cell of your Jupyter Notebook:
+
+```python
+pip install cmcrameri
+```
+
+Then we need to import all the necessary packages in our Jupyter Notebook.
+
+```python
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cmcrameri.cm as cmc
+import pandas as pd
+```
+
+In case some packages cannot be imported you need to install them first. For example if cmcrameri is missing and you get an error, type and run `pip install cmcrameri` in a new notebook cell, then re-execute the cell in your Jupyter Notebook.
+
+## Open and read metadata
+
+The netCDF dataset can now be opened with Xarray:
+
+```python
+dset = xr.open_dataset("data/CAMS-PM2_5-20211222.netcdf")
```

Comment:
hmmm it seems there is something lacking before this step to import the data no?